hw/top_earlgrey/ip/xbar_main/dv/autogen/xbar_cov_excl.el

```diff
@@ -0,0 +1,25 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// xbar_cov_excl.el generated by `tlgen.py` tool
+
+ANNOTATION: "[NON_RTL]"
+MODULE: uvm_pkg
+Assert \uvm_reg_map::do_write .unnamed$$_0.unnamed$$_1 "assertion"
+Assert \uvm_reg_map::do_read .unnamed$$_0.unnamed$$_1 "assertion"
+
```

Comment:
minor nit: These new lines are a bit arbitrary. Probably get rid of them (or add one before each ANNOTATION line).

Reply:
Those with no new lines are generated in a loop. Added a new line in the loop as well. Thanks